src/fundus/logging/__init__.py

```diff
@@ -1,3 +1,22 @@
-from .logger import basic_logger
+import logging
 
-__all__ = ["basic_logger"]
+__all__ = ["set_log_level", "create_logger"]
+
+_loggers = []
+
+_stream_handler = logging.StreamHandler()
+_formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+_stream_handler.setFormatter(_formatter)
+
+
+def create_logger(name: str) -> logging.Logger:
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.ERROR)
+    logger.addHandler(_stream_handler)
+    _loggers.append(logger)
+    return logger
+
+
+def set_log_level(level: int):
+    for logger in _loggers:
+        logger.setLevel(level)
```

Comment:
This definitely works. But I wondered if such a setup is possible where each module can just call `logger.getLogger(__name__)` to get their module logger, just like in the python [logging guide](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial). I failed to get this to work but wanted to give the idea here. Maybe you tried this as well?

Reply:
Good idea. The only problem is see is how we set the log level for all Fundus logger at once? Setting them individually wouldn't be a problem, but I have no solution for how we would set the whole library to `DEBUG` without keeping track of the loggers.